src/game_board/module.js

```diff
@@ -62,4 +62,10 @@ module.exports = angular.module('cardshifter.gameBoard', [ngRoute, ngAnimate, se
         replace: true,
         template: require('../card_model/card_template.html')
     };
+  })
+  .filter('capitalize', function() {
```

Comment:
This is not the greatest of names considering the filter's functionality, any better ideas?